README.md

```diff
@@ -41,13 +41,32 @@ Formats the token amount.
 #### Parameters
 
 - `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
+- `options.commify`: whether the token amount should have comma separators
 - `options.digits`: the number of digits to display. Defaults to `2`.
 - `options.displaySign`: whether the sign (`-` or `+`) should be displayed for the amount.
 
 ### TokenAmount#format(options)
```

Comment:
Not sure, but this seems to be duplicated with above?

Reply:
Hmmm it does; @bpierre what's supposed to be here?

Reply:
Ah, it should be `TokenAmount#toString(options)` :smile: